rpc_jobs/rpc_gating.yml

```diff
@@ -0,0 +1,20 @@
+- job:
+    name: "Merge-Trigger-rpc-gating"
+    project-type: pipeline
+    concurrent: true
+    properties:
+      - rpc-gating-github
+      - build-discarder:
+          num-to-keep: 30
+    triggers:
+        - github
+    parameters:
+      - rpc_gating_params
+    pipeline-scm:
+      scm:
+        - git:
+            url: https://github.com/rcbops/rpc-gating
+            branches:
+              - "${RPC_GATING_BRANCH}"
```

Comment:
This `$` is weirding me out here ... is that correct? I see this snippet of code used elsewhere so it shouldn't break anything, but I do wonder if it's right. Also, presumably we can put anything here and it doesn't matter due to RE-1645?

Reply:
Both good questions. I'm reusing the existing pattern. RE-1645 is not confirmed today and needs to be verified, so I'd rather leave this here in case it actually works. It is a JJB variable so "${var}" is right to me?

Reply:
This isn't a JJB template, so `${}` will be interpreted by Jenkins, as an environment variable lookup. Parameters are set as env vars before the lookup, so it should work fine.